docs/external-secrets.md

```diff
@@ -0,0 +1,53 @@
+---
+title: External secrets
+description: Use an external secrets vault with n8n.
+contentType: howto
+---
+
+# External secrets
+
+!!! info "Feature availability"
+	* External secrets are available on Enterprise Self-hosted and Cloud plans.
+	* n8n supports Infisical and HashiCorp Vault. 
+	* n8n doesn't support [HashiCorp Vault Secrets](https://developer.hashicorp.com/hcp/docs/vault-secrets){:target=_blank .external-link}.
+
+You can use an external secrets vault to manage credentials for n8n.
+
+## Connect n8n to your secrets vault
+
+1. In n8n, go to **Settings** > **External Secrets**.
+1. Select your vault provider.
+1. Enter the credentials for your provider:
+	* Infisical: provide a **Service Token**. Refer to Infisical's [Service token](https://infisical.com/docs/documentation/platform/token){:target=_blank .external-link} documentation for information on getting your token. If you self-host Infisical, enter the **Site URL**.
+
+	!!! note "Infisical environment"
+		Make sure you select the correct Infisical environment when creating your token. n8n will load secrets from this environment, and won't have access to secrets in other Infisical environments.
```

Comment:
Infisical have changed how environments work and you can now have a token that has access to multiple environments but we don't currently support it. So it might be worth adding a note saying we only support service tokens that have access to a single environment.